src/dune_rules/install_rules.mli

```diff
@@ -9,3 +20,53 @@ val symlink_rules :
 (** Generate rules for [.dune-package], [META.<package-name>] files. and
     [<package-name>.install] files. *)
 val gen_project_rules : Super_context.t -> Dune_project.t -> unit Memo.t
+
+module Build_path_prefix_map_dyn : sig
+  val pair_to_dyn : Build_path_prefix_map.pair -> Dyn.t
+
+  val to_dyn : Build_path_prefix_map.map -> Dyn.t
+end
+
+module Inverse_map : sig
+  val invert_maps :
+       Build_path_prefix_map.map String.Map.t
+    -> Build_path_prefix_map.map String.Map.t
+end
+
+module Build_map : sig
+  type t = Build_path_prefix_map.map String.Map.t
+
+  val to_dyn : t -> Dyn.t
+
+  val build_context_map_full :
+       Super_context.t
+    -> Install.Entry.Sourced.t list Package.Name.Map.t
+    -> Build_path_prefix_map.map
+
+  val build_all_maps_full :
+       Super_context.t Context_name.Map.t
+    -> Install.Entry.Sourced.t list Package.Name.Map.t Context_name.Map.t
+    -> Build_path_prefix_map.map String.Map.t
+
+  val build_context_map :
+       Super_context.t
+    -> Install.Entry.Sourced.t list Package.Name.Map.t
+    -> Build_path_prefix_map.map
+
+  val build_all_maps :
```

Comment:
Where is this function being used outside of this module? If it isn't, it shouldn't be in the mli.